kilkari/src/main/java/org/motechproject/nms/kilkari/service/impl/SubscriptionServiceImpl.java

```diff
@@ -70,6 +71,9 @@
     private boolean allowMctsSubscriptions;
 
     @Autowired
+    private SubscriberService subscriberService;
```

Comment:
This feels like a circular dependency reference. Subscriber service already has a dependency on subscription service and now we are introducing a dependency the other way as well.